web/modules/custom/collection/collection.module

```diff
@@ -320,17 +321,23 @@ function collection_entity_extra_field_info() {
     'visible' => FALSE,
   ];
 
+  $display['read_more'] = [
```

Comment:
Just a side note: Instead of hardcoding we should go more and more with "DS way". I think such fields can be easily configured in DS instead of coding them. But this is just my thinking for the future, not for this specific PR.